src/server/server.ts

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
+import yaml from 'yaml';
+
+import { TeamData } from './network/network';
+
+// Load team data (name, multicast/broadcast address, port)
+// First check for the YAML config file (config.yaml)
+//   If this exists, load team data from it
+//   If it does not exist, load team data from the command line
+// @param args An array of strings containing the positional arguments from the command line
+// @param cb A callback function that will process the extracted team data
+export function loadTeamData(
+  args: string[],
+  cb: (teamData: TeamData[]) => void,
+) {
+  // If the config file exists read it in instead of reading from the command line
+  // Otherwise use the command line arguments
+  // If there are no command line arguments, or not a multiple of 3 command line arguments, throw an error
+  fs.readFile(
+    'config.yaml',
+    { encoding: 'utf8', flag: 'r' },
```

Comment:
`r` is the default when reading, and options can be just the encoding:

```suggestion
    'utf8',
```